src/app/modules/account/pages/spouse-info/spouse-info.component.ts

```diff
@@ -173,6 +173,9 @@ export class SpouseInfoComponent extends BaseForm implements OnInit, AfterViewIn
       valid = valid && this.removedSpouse.cancellationDate !== undefined
         && this.removedSpouse.cancellationDate !== null;
     }
+    if (this.removedSpouse.cancellationReason === 4){
```

Comment:
Please put the variable name here instead of the value "4".

Reply:
I've added the changes in my latest commit. Thanks Harry.